.github/workflows/release.yml

```diff
@@ -9,6 +9,9 @@ on:
       - maint
     types:
       - closed
+  # Allow manually triggering a release via a "Run workflow" button on the
+  # workflow's page:
+  workflow_dispatch:
```

Comment:
@yarikoptic You also need to update the `if:` line below: https://github.com/datalad/release-action/pull/51/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R21